dev-docs/analytics/concert.md

```diff
@@ -4,8 +4,21 @@ title: Concert
 description: Concert Analytics Adapter
 modulecode: concert
 enable_download: false
+prebid_member: false
+tcfeu_supported: true
```

Comment:
How can you support `TCF` if you don't have a vendor id? If consent cannot be checked for your vendor, then it's supported.

```suggestion
tcfeu_supported: false
```